### DISCUSSION TITLE
FHIR standards light on content

### DISCUSSION THREAD

**Discussion by alan-amx** [tewhatuora/api-standards#41] [General]
### Summary

A general rather than a specific comment but the FHIR standards seemed the least fleshed out of all the sections. As this is for health APIs I would have expected the FHIR standards to be the most prominent section. They come across as almost an afterthought and feel lost amongst some of the boilerplate content sections that proceed them.

### Which area of the standards does this apply to?

- [ ] Part A - API Concepts
- [ ] Part B - API Security
- [ ] Part C - API Design and Development
- [X] Part D - FHIR
- [ ] Community guidelines

**Comment by swithinfoote** [comment]
Hi @alan-amx - thanks for your feedback. This feels more like a discussion item so I will move to discussions to gather further feedback from standards participants.
Cheers.

**Comment by ChrisSquats** [comment]
Hi @alan-amx  as Thanks for sharing your thoughts on the FHIR standards section. I get where you're coming from—it's a key part of health APIs, and it should probably get more spotlight. Your feedback helps us fine-tune our documentation to better serve users like you.

If you've got any ideas on how we could improve that section, let us know!
